apps/core/src/hooks/useGetKioskContents.ts

```diff
@@ -74,25 +74,27 @@ async function getOriginByteKioskContents(address: string, client: SuiClient) {
 }
 
 async function getSuiKioskContents(address: string, client: SuiClient) {
-	const ownedKiosks = await getOwnedKiosks(client, address!);
+	const kioskClient = new KioskClient({
```

Comment:
If this is a stateful client, then we should pass this in and not re-create it every time. Although to be fair this might change if you rework `KioskClient` to not be both data fetching and construction (which you should do).